src/api.raml

```diff
@@ -832,6 +832,44 @@ traits:
                                 description: |
                                     Demonstration of the list being empty (the first page will always return a valid result).
                                 value: !include samples/digest-list/v1/empty.json
+    post:
```

Comment:
Given the ID is coming from outside, I'd rather just have `PUT`.

Reply:
I think the ID can be auto-generated, although coming it from outside is safer for idempotence. We could document here the PUT and PATCH and deprecate the POST.